src/V3Ast.cpp

```diff
@@ -1129,7 +1201,18 @@ void AstNode::dumpTreeFile(const string& filename, bool append, bool doDump, boo
                 dumpTree(*logsp);
                 editCountSetLast();  // Next dump can indicate start from here
             }
+        } else {
+            const std::unique_ptr<std::ofstream> treedotp{V3File::new_ofstream(filename + ".dot", append)};
+            if (treedotp->fail()) v3fatal("Can't write " << filename);
+            *treedotp << "digraph vTree{\n";
+            *treedotp << "\tgraph\t[label=\"" << filename + ".dot"  << "\",\n";
```

Comment:
I would consider adding `rankdir=LR`. Because the nodes tend to be wider than tall, it likely uses display space better.